canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
+
+## Changed
+
+Deprecate the usage of the `Store` trait.
```

Comment:
If it's really deprecated should be under `### Deprecated` section, but I believe the trait does not exists anymore, so it's not deprecated ("soon-to-be removed"), it's actually removed. Also, this should be a list item (so starting with a dash), therefore this would probably looks like:

```

## [Unreleased]

### Removed

- Remove `Store` trait
```